helix-view/src/editor.rs

```diff
@@ -441,6 +440,8 @@ pub fn get_terminal_provider() -> Option<TerminalConfig> {
     None
 }
 
+#[cfg(not(any(windows, target_arch = "wasm32")))]
```

Comment:
Looks like this was moved when resolving a merge conflict - it should be right above `get_terminal_provider`